server/resource_controller.go

```diff
@@ -290,6 +291,9 @@ func (u UnescapedJSONRenderer) Render(w http.ResponseWriter) (err error) {
 	data = bytes.Replace(data, []byte("\\u003e"), []byte(">"), -1)
 	data = bytes.Replace(data, []byte("\\u0026"), []byte("&"), -1)
 
+	// Convert "_id" to "id"
+	data = bytes.Replace(data, []byte("_id"), []byte("id"), -1)
```

Comment:
This probably seems tedious, but a search on just those three characters (`_id`) in the whole JSON document seems too broad.  It would find `_id` anywhere -- even in strings like `my_id`, etc.  To be as safe as possible, we should really search on `"_id"` (with the quotes), or better yet -- if we're guaranteed to be able to rely on the formatting -- `"_id":` or `"_id" :` (I don't know if Go renders a space before the `:` or not).